compose/gettingstarted.md

```diff
@@ -85,11 +85,10 @@ In your project directory, create a file named `Dockerfile` and paste the
 following:
 
 ```dockerfile
-FROM python:3.7-alpine
+FROM python:3
 WORKDIR /code
 ENV FLASK_APP=app.py
 ENV FLASK_RUN_HOST=0.0.0.0
-RUN apk add --no-cache gcc musl-dev linux-headers
```

Comment:
does it work if you install `libffi-dev` ?

```suggestion
RUN apk add --no-cache gcc libffi-dev musl-dev linux-headers
```

Reply:
For me everything is working with the defaults